package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "mattermost-plugin-legal-hold",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {}
+}
```

Comment:
Probably accidentally added only. Can be removed.

Reply:
I'll remove it in the commit but should it be ignored in the `.gitignore`?

Reply:
No. It's in the root directory and it might have been added when after accidentally ran npm install instead at the correct directories.

Reply:
@yuney-worx4you The Playwright pipeline failed with:
```
npm ERR! The `npm ci` command can only install with an existing package-lock.json or
npm ERR! npm-shrinkwrap.json with lockfileVersion >= 1. Run an install with npm@5 or
npm ERR! later to generate a package-lock.json file, then try again.
```

The package.json you deleted was not correct. You should delete `package-lock.json` at the root directory and not the [e2e/playwright/package-lock.json](https://github.com/mattermost/mattermost-plugin-legal-hold/pull/97/commits/8705b47126ea06bc143864863bf68a1d2dd3ba24). Please update and see if the pipeline passed without issue.